node/src/proxy_server/mod.rs

```diff
@@ -4503,6 +4492,9 @@ mod tests {
         let neighborhood_recording = neighborhood_recording_arc.lock().unwrap();
         let record_opt = neighborhood_recording.get_record_opt::<NodeRecordMetadataMessage>(0);
         assert_eq!(record_opt, None);
+        TestLogHandler::new().exists_log_containing(&format!(
+            "ERROR: {test_name}: A bad exit node lied to us about the DNS failure"
```

Comment:
Can you make this message a bit more specific? For example, we'd like to know that the issue is a stream key that doesn't exist. When we eventually get around to malefactor banning, each ban will probably have a message associated with it telling why the malefactor was banned; think about looking at a database table full of malefactor bans from months or years ago, and use a message that would make clear what the malefactor's crime was.

Reply:
@dnwiebe it appears this error message could be wrong.
Will bring this up in the Dev meeting.